templates/blog/tag.html

```diff
@@ -2,16 +2,23 @@
 
 {% block title %}Blog posts tagged "{{ tag.name }}"{% endblock %}
 
+{% set is_paper = True %}
+
 {% block content %}
-  <div class="p-strip--image is-light is-deep" style="background-image: url(https://assets.ubuntu.com/v1/f8a323a7-image-background-paper.png);">
-    <div class="row u-equal-height u-vertically-center">
-      <div class="col-10">
-        <h1 class="u-no-margin--bottom">Blog posts tagged "{{ tag.name }}"</h1>
+  <div class="p-strip">
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-4 col-start-medium-3">
+        <div class="row p-section">
```

Comment:
Again nested rows

Reply:
see above, this is how the grid is documented to work: https://vanillaframework.io/docs/patterns/grid#nested-columns